coldfront/core/utils/context_processors.py

```diff
@@ -73,3 +83,34 @@ def primary_cluster_name(request):
     return {
         'PRIMARY_CLUSTER_NAME': settings.PRIMARY_CLUSTER_NAME,
     }
+
+
+def request_alert_counts(request):
+
```

Comment:
I think these computations will end up being run for all users, no?

You should be able to check `request.user.is_superuser`/`.is_staff` here.

Reply:
> I'd also only run the query if the relevant flag is enabled -- see the template.

Sorry, clarifying what I meant here, because it was unclear:

Some of these requests are only relevant when certain feature flags are enabled. In the template, flags are used to show/hide some request types. The corresponding count need not be calculated if the request type isn't going to be displayed.

Reply:
Got it. Done!